README.md

```diff
@@ -10,20 +10,20 @@ Each one has it's own folder and README describing the package.
-| -------------------------- | ----------------------------------------------- | -------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
-| @itwin/tree-widget-react   | [tree-widget](./packages/itwin/tree-widget/)    | [changelog](./packages/itwin/tree-widget/CHANGELOG.md)   | [![npm version](https://badge.fury.io/js/%40itwin%2Ftree-widget-react.svg)](https://badge.fury.io/js/%40itwin%2Ftree-widget-react)     |
-| @itwin/property-grid-react | [property-grid](./packages/itwin/property-grid) | [changelog](./packages/itwin/property-grid/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40itwin%2Fproperty-grid-react.svg)](https://badge.fury.io/js/%40itwin%2Fproperty-grid-react) |
-| @itwin/measure-tools-react | [measure-tools](./packages/itwin/measure-tools) | [changelog](./packages/itwin/measure-tools/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40itwin%2Fmeasure-tools-react.svg)](https://badge.fury.io/js/%40itwin%2Fmeasure-tools-react) |
+| Name                         | Folder                                                | Changelog                                                    | Version                                                                                                                                        |
+| ---------------------------- | ----------------------------------------------------- | ------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| @itwin/tree-widget-react     | [tree-widget](./packages/itwin/tree-widget/)          | [changelog](./packages/itwin/tree-widget/CHANGELOG.md)       | [![npm version](https://badge.fury.io/js/%40itwin%2Ftree-widget-react.svg)](https://badge.fury.io/js/%40itwin%2Ftree-widget-react)             |
+| @itwin/property-grid-react   | [property-grid](./packages/itwin/property-grid)       | [changelog](./packages/itwin/property-grid/CHANGELOG.md)     | [![npm version](https://badge.fury.io/js/%40itwin%2Fproperty-grid-react.svg)](https://badge.fury.io/js/%40itwin%2Fproperty-grid-react)         |
+| @itwin/measure-tools-react   | [measure-tools](./packages/itwin/measure-tools)       | [changelog](./packages/itwin/measure-tools/CHANGELOG.md)     | [![npm version](https://badge.fury.io/js/%40itwin%2Fmeasure-tools-react.svg)](https://badge.fury.io/js/%40itwin%2Fmeasure-tools-react)         |
+| @itwin/breakdown-trees-react | [breakdown-trees](./packages/bentley/breakdown-trees) | [changelog](./packages/bentley/breakdown-trees/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fbreakdown-trees-react.svg)](https://badge.fury.io/js/%40bentley%2Fbreakdown-trees-react) |
```

Comment:
pkg src needs to move into the itwin directory

the badges should point to itwin not bentley